viral/ensemble_reactivation.py

```
@@ -0,0 +1,861 @@
+        interpolation="none",
+    )
+    # desired_ratio = raster.shape[1] / raster.shape[0]
+    # ax3.set_aspect(desired_ratio / 5)  # Reduce the stretching
+
+    ax3.set_ylabel("Place cell ID")
+    ax3.set_xlabel("Frames")
+
+    # ax3.set_title("Smoothed offline firing rate raster")
+
+    plt.savefig("plots/grosmark_panel.svg", dpi=300)
+    1 / 0
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)